package.json

```diff
@@ -8,24 +8,22 @@
     "url": "https://github.com/latticexyz/mud.git"
   },
   "scripts": {
+    "// TODO make this use turob instead?  an we get rid of lerna?  link:packages": "lerna exec 'pnpm link --global'",
+    "// TODO what is this one?   Can we get rid of lerna?  entry:dist": "lerna run prepack",
+    "// TODO what is this one?   Can we get rid of lerna?  entry:src": "lerna run postpack",
+    "// TODO what is this one?release:yalc": "(pnpm entry:dist && lerna exec npx yalc push); pnpm entry:src",
     "build": "turbo run build",
     "clean": "turbo run clean",
     "dev": "turbo run dev --concurrency 100",
     "docs": "pnpm recursive run docs && pnpm prettier --write '**/*.md' && pnpm run --filter docs build:prepare && pnpm run --filter docs build",
-    "entry:dist": "lerna run prepack",
-    "entry:src": "lerna run postpack",
     "foundryup": "curl -L https://foundry.paradigm.xyz | bash && bash ~/.foundry/bin/foundryup",
     "gas-report": "pnpm recursive run gas-report",
-    "link:packages": "lerna exec 'pnpm link --global'",
     "lint": "pnpm prettier:check && eslint . --ext .ts --ext .tsx",
     "prepare": "husky install && (forge --version || pnpm foundryup)",
     "prettier": "prettier --write '**/*.{ts,tsx,css,md,mdx,sol}'",
     "prettier:check": "prettier --check '**/*.{ts,tsx,css,md,sol}'",
-    "release": "pnpm install && lerna publish --no-private --force-publish",
-    "release:canary": "lerna publish premajor --preid alpha.1 --canary --no-private --force-publish",
-    "release:ci": "lerna publish --no-private --force-publish --yes",
-    "release:manual": "pnpm --recursive --no-bail run release",
-    "release:yalc": "(pnpm entry:dist && lerna exec npx yalc push); pnpm entry:src",
+    "release": "pnpm install && changeset publish",
```

Comment:
do we need to run `pnpm build` here too (to create the dist files that are not on github but should be on npm) or does changeset do that internally by default?

Reply:
Yes meant to add it here.   can do it in the GitHub action too with ‘with: build’ but feels better here just in case this gets ran manually